public/api/compliance/v1/openapi.json

```diff
@@ -0,0 +1 @@
+../../../../swagger/v1/openapi.json
```

Comment:
Are there any implication (impact) of creating `public/api/compliance/v1/` directory structure, in regards to authentication and API routing?
Would rails follow a symlink in a production env to serve the file?

Reply:
1. I tested all the possible endpoints that could hit the directory structure and have not seen any differences between before/after
2. yes, otherwise the CI below would not pass as the JSON is being polled as a `livenessProbe`